gammapy/makers/safe.py

```diff
@@ -40,6 +40,12 @@ class SafeMaskMaker(Maker):
         the `aeff_percent` or `bias_percent` are computed.
     offset_max : str or `~astropy.units.Quantity`
         Maximum offset cut.
+    n_95percentile : float
```

Comment:
What is the advantage of hard-coding to `95%` and then scale up? Maybe we just take the value of percentile directly? And maybe mention `bkg` in the name, e.g like `bkg_clip_percentile`

Reply:
It very hard to find the exact percentile that corresponds to the good-to-bad values transition, but we can reasonably assume that bad values are within the 5 last percentile and much larger than the good ones so I opted for that solution.